src/main/java/util/RandomSymbolUtil.java

```diff
@@ -0,0 +1,27 @@
+package util;
+
+import com.github.javafaker.Faker;
+
+/***
+ * Random generator of strings
+ */
+public class RandomSymbolUtil {
+
+    static Faker faker = new Faker();
+
+    public static String getRandomLogin() {
```

Comment:
please add java docs to all the methods here

Reply:
done